controller/array_action/array_mediator_svc.py

```diff
@@ -174,6 +174,11 @@ def get_volume_name(self, volume_id):
         # TODO: CSI-1024
         pass
 
+    def is_volume_has_snapshots(self, volume_id):
+        volume_name = self._get_vol_by_wwn(volume_id)
+        filter_value = 'source_vdisk_name={0}'.format(volume_name)
```

Comment:
do we prefer this over `source_vdisk_id`, to save `raise VolumeNotFoundError` logic duplication?
we should probably avoid sending unnecessary extra commands wherever possible, which is another reason for having a generic `get_cli_object` method

Reply:
We don't have volume id. We use wwn as id.

Reply:
oh. then we should just consider renaming the interface parameter from `volume_id` to `volume_wwn`

Reply:
don't want to think about it now
not relevant here at all

Reply:
I know, that's why I wrote "in a separate ticket".
I added it as an option in CSI-1516